File: src/main/java/com/sfh/pokeRogueBot/model/enums/Stat.java
package com.sfh.pokeRogueBot.model.enums;

public enum Stat {

    HP,
    ATK,
    DEF,
    SPATK,
    SPDEF,
    SPD;

    public static String getBaseStatBoosterItemName(Stat stat) {
        switch (stat) {
            case HP:
                return "HP Up";
            case ATK:
                return "Protein";
            case DEF:
                return "Iron";
            case SPATK:
                return "Calcium";
            case SPDEF:
                return "Zinc";
            case SPD:
                return "Carbos";
        }
        return null;
    }

}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/enums/Stat.kt
package com.sfh.pokeRogueBot.model.enums

enum class Stat {
    HP,
    ATK,
    DEF,
    SPATK,
    SPDEF,
    SPD;

    companion object {
        fun getBaseStatBoosterItemName(stat: Stat): String? {
            return when (stat) {
                HP -> "HP Up"
                ATK -> "Protein"
                DEF -> "Iron"
                SPATK -> "Calcium"
                SPDEF -> "Zinc"
                SPD -> "Carbos"
            }
        }
    }
}